### DISCUSSION TITLE
Dormitory & Student accommodation

### DISCUSSION THREAD

**Discussion by debyos** [osmandapp/OsmAnd#11584] [Maps Related]
Dear All,

OsmAnd is a great tool I use (and also translate) almost on a daily basis. (And I even have a LIVE subscription.) 
Thus, thank you for the development.

# 🚀 Rendering dormitories / student accommodations
### Description
[`building=dormitory`](https://wiki.openstreetmap.org/wiki/Tag%3Abuilding%3Ddormitory) is a popular tag with about 40.000 entries on OpenStreetMap.

[`amenity=student_accommodation`](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dstudent_accommodation) is less popular but existing and might become better known in the future. There are dozens of tags with far less entries on OSM which can be found in OsmAnd.

Both tags describe essentially the same feature so it would be wise to have both of them in OsmAnd.

### The solution
They appear in OsmAnd like thousands of other tags.

### Possible alternative
Hm… Well… The case remains as it is and they do not appear / are not rendered.

**Comment by vshcherb** [comment]
It's a bit unclear how it should be rendered, bear your mind that rendering is not a POI icon and won't have additional information displayed as hotel / hostel.

**Reply by WaleedMortaja** [reply]
I was searching for a building that I can see on openstreetmap website, but was impossible to find in osmand even when searching the exact name to the letters.
The building was tagged as `amenity=student_accommodation`. It also have a name and operator. There is no `building` tag.

Then I found this issue/discussion. I do not know about rendering and icon, but if the name of the place appears in the center of its area, and is searchable, that would be a huge progress.
